### PR TITLE
Fireaxe Removes Plating (Delta-V#1700), Maintenance Jack

### PR DESCRIPTION
**Pull request by whatston3**
## About the PR

Cherry-picks [Delta-V#1700](https://github.com/DeltaV-Station/Delta-v/pull/1700) (Thank you, MilonPL).

Adds a maintenance jack as a purchasable tool from the EngiVend machine that uses the new Axing action.  It's a huge item that's carriable on your back or suit storage (though it currently lacks sprites for on-body states).

Adds yellow crowbars to the POI engivend inventory.

## Why / Balance

Convenience, nice to not have to use RCD ammo to remove ship plating.

## How to test

1. Visit the EngiVend, purchase a maintenance jack.
2. Spawn a crowbar.
3. On Frontier or the bus, remove the floor tiling with the crowbar.  Try to remove the plating with the maintenance jack.  You should get a do-after, but nothing should happen.
4. Warp to a POI.
5. Remove the floor tiling with a crowbar, remove the plating with the maintenance jack.  It should remove it, and you should see the bare lattice exposed underneath.

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

None.

**Changelog**

:cl:
- add: The fireaxe can now remove hull plating.
- add: A new tool, the maintenance jack, is available from the EngiVend. It can remove hull plating.

### CODE REVIEW

File: Resources/Prototypes/_NF/Entities/Objects/Tools/tools.yml [hunk 1]
```diff
@@ -29,3 +29,37 @@
...
+    quickEquip: false
+    slots:
+    - back
+    - suitStorage
+  - type: Tool
+    qualities:
+      - Axing
+  - type: ToolTileCompatible
+  - type: UseDelay
+    delay: 1
```
Comment (dvir001) [inline Resources/Prototypes/_NF/Entities/Objects/Tools/tools.yml:65]: Wont sitters use this to fast remove floors in frontier?

Comment (whatston3) [reply Resources/Prototypes/_NF/Entities/Objects/Tools/tools.yml:65]: No, it's grid-protected.  It'll space POIs, though.

File: Resources/Textures/_NF/Objects/Tools/maintenance-jack.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Based on maintenance-jack.dmi from NebulaSS13, modified by Whatstone",
```
Comment (whatston3) [inline Resources/Textures/_NF/Objects/Tools/maintenance-jack.rsi/meta.json:4]: Need to reference the commit where I took it, just marking it as 413dc17c9757bf33984534a4d9b3e3db538a6285.

**Review by ErhardSteinhauer — APPROVED** [review]
Works.

**Comment by dvir001** [comment]
I can finally say "time to jack off the tiles"
10/10 will merge again.

**Comment by Tych0theSynth** [comment]
> I can finally say "time to jack off the tiles" 10/10 will merge again.

banned

**Pull request merged**